test/system/030-run.bats

```diff
@@ -1399,4 +1399,28 @@ search               | $IMAGE           |
     run_podman rm -f -t0 $cid
 }
 
+@test "podman run with mounts.conf missing" {
+    skip_if_remote "--default-mounts-file is not supported for remote clients"
+    MOUNTS_CONF=$PODMAN_TMPDIR/mounts.conf
+    run_podman run --rm --default-mounts-file=${MOUNTS_CONF} $IMAGE echo test1
+    assert "$output" = "test1" "No warning messages on missing mounts file"
+
+    touch ${MOUNTS_CONF}
+
+    run_podman run --rm --default-mounts-file=${MOUNTS_CONF} $IMAGE echo test2
+    assert "$output" = "test2" "No warning messages on empty mounts file"
+
+    echo /tmp/bogus > $PODMAN_TMPDIR/mounts.conf
+    run_podman run --rm --default-mounts-file=${MOUNTS_CONF} $IMAGE echo test3
+    assert "$output" = "test3" "No warning messages on missing content in mounts file"
+
+    randfile=$(random_string 30)
+    randcontent=$(random_string 30)
+    mkdir -p $PODMAN_TMPDIR/mounts
+    echo $randcontent > $PODMAN_TMPDIR/mounts/$randfile
+    echo $PODMAN_TMPDIR/mounts:/run/secrets > $PODMAN_TMPDIR/mounts.conf
```

Comment:
Could use `${MOUNTS_CONF}` instead of `$PODMAN_TMPDIR/mounts.conf` for consistency.  Same for the other reference above.